cmd/README.md

```diff
@@ -74,6 +74,14 @@ specify the passphrase with the ``--passphrase`` flag.
 This command is only able to import the wallet from which you can re-generate
 your key pairs.
 
+## List registered wallet
+
+If you want to list all the registered wallets, use the following command:
```

Comment:
Does this mean all the keypairs registered with this wallet? Or does it mean something else?

Reply:
That just mean the wallets. Not their keys. For the keys, use `key list`

Reply:
So this is for wallets within the wallet?

Reply:
The go-wallet project is able to manage several wallets. And each wallet has one to several keys.